### ISSUE TITLE
Should Interceptors depend on EJB API?

### ISSUE THREAD

**Issue by ljnelson** [jakartaee/specifications#115]
I note that the Interceptors API jar requires the EJB API jar:
```
[snip from Maven-related experiments involving dependency:tree]
[INFO] |  +- jakarta.interceptor:jakarta.interceptor-api:jar:1.2.5:provided
[INFO] |  |  +- jakarta.annotation:jakarta.annotation-api:jar:1.3.5:provided
[INFO] |  |  \- jakarta.ejb:jakarta.ejb-api:jar:3.2.6:provided
```
Was that intentional?  `InvocationContext#getTimer()` returns `Object`, and the Javadoc reference to `{@link javax.ejb.Timer}` [isn't linked anyway](https://jakarta.ee/specifications/interceptors/1.2/apidocs/javax/interceptor/InvocationContext.html#getTimer--).  Can this API jar dependency be removed?

**Comment by ljnelson** [comment]
See also: https://github.com/eclipse-ee4j/interceptor-api/issues/31

**Comment by keilw** [comment]
If it's just in the JavaDoc and there is no dependency whatsoever, not even an optional one, then it could be removed I guess. And the reference ideally should be removed and  `{@link javax.ejb.Timer}` replaced e.g. with `{@code javax.ejb.Timer}` which has the same formatting effect.

**Comment by kwsutter** [comment]
Since this looks to be resolved via [eclipse-ee4j/interceptor-api#31](https://github.com/eclipse-ee4j/interceptor-api/issues/31), I will close this Issue.
